modules/standard/FileSystem.chpl

```diff
@@ -894,7 +971,7 @@ private module GlobWrappers {
 
    :yield: The matching filenames as strings
 */
-iter glob(pattern: string = "*"): string {
+iter glob(pattern: bytes = b"*"): bytes {
```

Comment:
you'll have to update the docs, but this one I expect to always return bytes, no matter if pattern is string or bytes